scale/storage/serializers.py

```diff
@@ -117,4 +117,11 @@ class ScaleFileBaseSerializer(ModelIdSerializer):
 
 class ScaleFileSerializer(ScaleFileBaseSerializer):
     """Converts Scale file model fields to REST output"""
-    workspace = WorkspaceSerializer()
+
+    file_type = serializers.CharField()
+    data_type = serializers.CharField()
```

Comment:
Is this correct? data_type is defined differently in the ScaleFileBaseSerializer.